index.js

```diff
@@ -23,28 +33,41 @@ module.exports = bundler => {
 			return;
 		}
 		console.log('');
+
 		const spinner = ora(chalk.grey('Fixing og:image link')).start();
 		const start = Date.now();
 
 		const htmlPath = path.join(bundler.options.outDir, 'index.html');
 		const html = fs.readFileSync(htmlPath).toString();
+		let hasErrors = false;
 
-		const ogImageTag = getMetaTag(html, 'og:image');
-		const ogImageContent = getMetaTagContent(ogImageTag);
+		try {
+			const ogImageTag = getMetaTag(html, 'og:image');
+			const ogImageContent = getMetaTagContent(ogImageTag, 'og:image');
 
-		const ogUrlTag = getMetaTag(html, 'og:url');
-		const ogUrlContent = getMetaTagContent(ogUrlTag);
+			const ogUrlTag = getMetaTag(html, 'og:url');
+			const ogUrlContent = getMetaTagContent(ogUrlTag, 'og:url');
 
-		const absoluteOgImageUrl = url.resolve(ogUrlContent, ogImageContent);
-		const ogImageTagAbsoluteUrl = ogImageTag.replace(ogImageContent, absoluteOgImageUrl);
-		const patchedHtml = html.replace(ogImageTag, ogImageTagAbsoluteUrl);
+			const absoluteOgImageUrl = url.resolve(ogUrlContent, ogImageContent);
+			const ogImageTagAbsoluteUrl = ogImageTag.replace(ogImageContent, absoluteOgImageUrl);
+			const patchedHtml = html.replace(ogImageTag, ogImageTagAbsoluteUrl);
 
-		fs.writeFileSync(htmlPath, patchedHtml);
+			fs.writeFileSync(htmlPath, patchedHtml);
+		} catch (error) {
+			spinner.fail(error.message);
+
+			hasErrors = true;
+		}
 
 		const end = Date.now();
+		const symbol = hasErrors ? chalk.red('✖') : '✨ ';
+		const text = hasErrors ?
+			chalk.red('Failed to fix og:image link. Please fix the above error.') :
```

Comment:
Maybe remove the `Please fix the above error.` text here.

This could be triggered by `fs.writeFileSync` and it might not make sense to say fix whatever the output of that is.

Just `Failed to fix og:image link.` should be fine. If there is a clear error above it the user will know that's what needs attention.